src/layout/Content/content.util.js

```diff
@@ -0,0 +1,33 @@
+export const generateGroupedItems = (content, selectedRowData) => {
+  const groupedItems = {}
+
+  content.forEach(contentItem => {
+    if (selectedRowData && selectedRowData[contentItem.name]?.content) {
```

Comment:
The new `?.` operator is not alone.
It has two friends: `?.[]` and `?.()`.
One for accessing properties with brackets notation and one for calling functions.
So you can do:
```suggestion
    if (selectedRowData?.[contentItem.name]?.content) {
```